### DISCUSSION TITLE
Undefined symbols for architecture x86_64 when installing via MacPorts

### DISCUSSION THREAD

**Discussion by 0xd3e** [extrawurst/gitui#1552] [Q&A]
Hi,

the MacPorts users are having the issue that gitui cannot be built and results in the error "Undefined symbols for architecture x86_64". This seems to be related to OpenSSL.

The full error including the command that failed can be seen here: https://github.com/macports/macports-ports/pull/17732

I'd appreciate any help that points me to the cause. I'm not a Rust developer (yet).

**Comment by extrawurst** [comment]
I am not using macports myself. Probably have to wait for @herbygillot - does it work for you using the brew recipe?

**Comment by 0xd3e** [comment]
I fixed it be using the `vendor-openssl` feature. (https://github.com/macports/macports-ports/pull/17732)
